Module1/Calculator/Calculator_with_violations_principles/src/com/company/CalculatorViolations/Calculator.java

```diff
@@ -15,42 +17,34 @@ public static void main(String[] args) {
         double inputX, inputY;
         double answer = 0;
         char operation;
-        InputStream input =  System.in;
-        Scanner sc = new Scanner(input);
         MessengerI messenger = new FrenchMessenger();
-        try {
+        try (Scanner sc = new Scanner(System.in)){ //try with resource
             messenger.inputFirstNumMessage();
             inputX = sc.nextDouble();
             messenger.inputOperationMessage();
             operation = sc.next().charAt(0);
             messenger.inputSecondNumMessage();
             inputY = sc.nextDouble();
-            BaseOperations ops = new BaseOperations() {
-            };
             //DRY
             switch (operation) {
```

Comment:
instead of using of switch you can add method to enum which return corresponding element by symbol